TourMate/TourMate/Models/DomainModels/Accommodation.swift

```diff
@@ -15,13 +15,14 @@ struct Accommodation: Plan {
     var name: String = "Accommodation"
     var startDate: Date
     var endDate: Date
-    var timeZone: TimeZone
-    var imageUrl: String
+    var startTimeZone: TimeZone
+    var endTimeZone: TimeZone?
+    var imageUrl: String?
```

Comment:
I see we have start and end time zones now, should we also add it into the data models?